### DISCUSSION TITLE
Outdated Baresip version in Ubuntu/Debian repositories

### DISCUSSION THREAD

**Discussion by haricane8133** [baresip/baresip#2890] [Q&A] [answered]
When I install baresip in my Raspberry Pi with the following command,

`sudo apt install baresip`

v.1.0.0 from 2020 gets installed. Are there plans to push the latest baresip package?

This would be really helpful as the manual builds are continuously failing in my device.

**Comment by sreimers** [comment, marked as answer]
We're not responsible for the official debian packages, they are maintained by debian itself. If you wan't to build manual, you have to remove all old dependencies (libre, librem) too and follow the https://github.com/baresip/baresip/wiki/Install:-Stable-Release guide, if any build errors occur, we need the build logs.

**Comment by haricane8133** [comment]
I was able to figure out the issue with manual build...
I had to install `libssl-dev` and `zlib1g-dev` to get the lib paths set properly

However, I would like to know how we can turn on the module pipewire instead of alsa.
I will start a new discussion.
